### DISCUSSION TITLE
XMP deletion after indexing

### DISCUSSION THREAD

**Discussion by richardoswald** [photoprism/photoprism#3179] [Q&A]
Hello I'm in the middle of a large Apple Photos migration (just over 80k photos) to Photoprism. I saw from the [documentation](https://docs.photoprism.app/user-guide/use-cases/apple/) it was recommended to export XMP files for the original unmodified photos.

I was wondering if Photoprism somehow embeds the XMP data into the images after indexing so I can remove the sidecar files when indexing is complete? Not sure if this is possible or recommended, but it would save me from another 80k of files and make things cleaner.

**Comment by graciousgrey** [comment]
PhotoPrism does not yet write metadata from xmp or other sources to the original image files. We do however write most metadata to human readable yml files in case the functionality is not disabled.

https://docs.photoprism.app/user-guide/library/metadata/#export
